File: cmd-tools/go/popcount.go
package main

import (
	"fmt"
	"log"
	"strconv"
	"strings"
)

var pc [256]byte

func init() {
	for i := range pc {
		pc[i] = pc[i/2] + byte(i&1)
	}
}

func popCount(n uint64) int {
	return int(pc[byte(n>>(0*8))] +
		pc[byte(n>>(1*8))] +
		pc[byte(n>>(2*8))] +
		pc[byte(n>>(3*8))] +
		pc[byte(n>>(4*8))] +
		pc[byte(n>>(5*8))] +
		pc[byte(n>>(6*8))] +
		pc[byte(n>>(7*8))])
}

func readPopCount(args []string) {
	for _, l := range args {
		l = strings.TrimSpace(l)
		num, err := strconv.ParseUint(l, 10, 64)
		if err != nil {
			log.Fatalf("Invalid Input: %v", err)
		}
		fmt.Printf("%d - Pop Count: %d\n", num, popCount(num))
	}
}

File: cmd-tools/python/popcount.py
def popCount(n: int) -> int:
    pc = [0] * 256
    for i in range(256):
        pc[i] = pc[i // 2] + (i & 1)

    return (pc[n & 0xFF] + pc[(n >> 8) & 0xFF] + pc[(n >> 16) & 0xFF] +
            pc[(n >> 24) & 0xFF] + pc[(n >> 32) & 0xFF] +
            pc[(n >> 40) & 0xFF] + pc[(n >> 48) & 0xFF] + pc[(n >> 56) & 0xFF])


def readPopCount(args: list[str]):

    for line in args:
        line = line.strip()
        try:
            num = int(line)
            print(f"{num} - Pop Count: {popCount(num)}")
        except ValueError as err:
            print(f"Invalid Input: {err}")
